spring-web/src/main/java/org/springframework/http/client/reactive/JdkClientHttpRequest.java

```diff
@@ -112,6 +113,15 @@ protected void applyCookies() {
 				.flatMap(List::stream).map(HttpCookie::toString).collect(Collectors.joining(";")));
 	}
 
+	/**
+	 * Not implemented, since {@link HttpRequest} does not offer any possibility to add request attributes
+	 */
+	@Override
+	protected void applyAttributes() {
+		// TODO
+		throw new RuntimeException(String.format("Using attributes is not available for %s", HttpRequest.class.getName()));
```

Comment:
Can somebody please verify if there is an option to set request attributes for the `java.net.http.HttpRequest`?
If there is not, what should be the proper behaviour, should an Exception be thrown to the user if `applyAttributes` is set to `true`? Or should it simply always default to `false`?

Reply:
If the boolean flag is removed as suggested above, this should become a no-op.